apis/apis.json

```diff
@@ -507,30 +507,30 @@
     "id": "Google.Cloud.Spanner.V1",
     "productName": "Google Cloud Spanner",
     "productUrl": "https://cloud.google.com/spanner/",
-    "version": "1.1.0-beta01",
+    "version": "1.1.0-beta02",
```

Comment:
I don't think this beta is meant to be updated yet?

Reply:
Well spotted. Will revert that one - but in a separate commit as doing anything else will be too complicated :)

Reply:
SGTM :)